File: Projects/Project_1/Grass.cpp
#include "Grass.h"

Grass::Grass(int whenWasBorn, Coordinates coords, World* world) : Plant(0, whenWasBorn, 'g', coords, world, Organism::TypeOfOrganism::GRASS, false) {

}

void Grass::GetOrganismName() {
	cout << "Grass";
}

Grass::~Grass(){}

File: Project_2-Java/po_projekt2/organisms/plants/Grass.java
package po_projekt2.organisms.plants;

import java.awt.Color;
import po_projekt2.World;
import po_projekt2.organisms.Plant;
import po_projekt2.worldElements.Coordinates;

/**
 *
 * @author Bogna
 */
public class Grass extends Plant{
    
    public Grass(int whenWasBorn, Coordinates coords, World world){
        super(0, whenWasBorn, new Color(0,255,127), "Grass", coords, world, typeOfOrganism.GRASS, false);
    }
}
